maven-plugin/src/main/java/org/openmrs/maven/plugins/ServerUpgrader.java

```diff
@@ -54,6 +54,9 @@ public void upgradeToDistro(Server server, Distribution distribution, boolean ig
 				return;
 			}
 		}
+
+		parentTask.distroHelper.validateDistribution(distroProperties);
```

Comment:
Per SDK-365, this moves the validation earlier in the process.  Previously this was not running until after war/module/etc changes had already been applied to the server.